README.md

```diff
@@ -141,7 +137,8 @@ Parameter | Description | Default
 `keys.repository` | Keyserver container image repository | `cscfi/ega-keyserver`
 `keys.imageTag` | Keyserver container image version | `"1.0"`
 `keys.imagePullPolicy` | Keyserver container image pull policy | `IfNotPresent`
-`keys.port` | Keyserver port | `8443`
+`keys.port` | Keyserver port | `8080`
+`keys.servicePort` | Keyserver port | `9095`
```

Comment:
```suggestion
`keys.servicePort` | Keyserver Service port | `9095`
```

we need to emphasize what port is used for what ... see if any other services suffer from the same miss in documentation